mtb-mw-dependencies-manifest.xml

```diff
@@ -928,4 +964,68 @@
       </version>
     </versions>
   </depender>
+  <depender>
+    <id>cy-mbedtls-acceleration</id>
+    <versions>
+      <version>
+        <commit>latest-v1.X</commit>
+        <dependees>
+          <dependee>
+            <id>mbedtls</id>
+            <commit>mbedtls-2.26.0</commit>
```

Comment:
One comment: cy-mbedtls-acceleration is not fully integrated into MTB. We dont have a mbedtls-2.26.0 in our mbedtls asset (the most recent one we pick is 2.25.0). According to the cy-mbedtls-acceleration documentation, they expect mbedtls to be git cloned manually. Therefore, there is no need to have this entry. And having this entry as a dependee results in a warning. Perhaps, leave this entry out from the dependee list